tooling/bundler/src/bundle/windows/templates/main.wxs

```diff
@@ -111,6 +92,19 @@
                 <RegistryKey Root="HKCU" Key="Software\\{{manufacturer}}\\{{product_name}}">
                     <RegistryValue Name="InstallDir" Type="string" Value="[INSTALLDIR]" KeyPath="yes" />
                 </RegistryKey>
+                {{#each deep_link_protocols as |protocol| ~}}
+                                                                                                    <!-- Change the Root to HKCU for perUser installations -->
```

Comment:
formatting and just to not sound as a broken record, can we revert most of these formatting changes?